plugins/Diagnostics/Diagnostic/CronArchivingLastRunCheck.php

```diff
@@ -90,8 +91,13 @@ public function execute()
 
     private function getArchivingCommand()
     {
-        $domain = Config::getHostname();
-        return PIWIK_INCLUDE_PATH . '/console --matomo-domain=' . $domain . ' core:archive';
+        if (Url::isValidHost()) {
+            $domain = Config::getHostname($checkIfTrusted = true);
```

Comment:
doesn't `isValidHost()` actually automatically check if the host is trusted (if not disabled in config)? Wondering if we still need the changes in `Config::getHostname()`

Reply:
ok